airbyte-webapp/src/components/ui/ListBox/ListBox.tsx

```diff
@@ -59,6 +67,11 @@ export const ListBox = <T,>({
                 )}
               </Listbox.Option>
             ))}
+            {footer && (
```

Comment:
Also not sure if the footer should go here, or somehow be included in the options array. This made more sense to me while coding this up."